tests/Behat/features/archive-block-element.feature

```diff
@@ -24,7 +24,8 @@ Feature: Archive elements in the CMS
     Given I see a list of blocks
     When I press the "View actions" button
       Then I should see the archive button for block 1
-    When I press the "Archive" button
+    # Use a css selector instead of the "Archive" button otherwise it will get confused with the Archive ModelAdmin
+    When I click on the ".element-editor__actions-archive.dropdown-item" element
```

Comment:
Just for context, `I press the "Archive" button` is kind of dumb because it doesn't distinguish between buttons and link.

Fixing it in behat-testing could be difficult difficult because it could potentially break a bunch of other things in all the other module. And it's not such a problem since it's relatively uncommon to have a link and a button on the same page with the same label.